com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -283,7 +284,6 @@ Shader "Universal Render Pipeline/Baked Lit"
             #pragma multi_compile _ LIGHTMAP_ON
             #pragma multi_compile_fog
             #pragma multi_compile_instancing
-            #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
```

Comment:
should this be removed?

Reply:
Yes, it accidentally had that line twice. 
In lines 282 and 286. Removed the one in 286.